node/index.js

```diff
@@ -29,171 +32,179 @@ var PLAID_COUNTRY_CODES = envvar.string('PLAID_COUNTRY_CODES', 'US').split(',');
 // that the bank website should redirect to. You will need to configure
 // this redirect URI for your client ID through the Plaid developer dashboard
 // at https://dashboard.plaid.com/team/api.
-var PLAID_REDIRECT_URI = envvar.string('PLAID_REDIRECT_URI', '');
+const PLAID_REDIRECT_URI = process.env.PLAID_REDIRECT_URI || "";
 
 // Parameter used for OAuth in Android. This should be the package name of your app,
 // e.g. com.plaid.linksample
-var PLAID_ANDROID_PACKAGE_NAME = envvar.string('PLAID_ANDROID_PACKAGE_NAME', '');
+const PLAID_ANDROID_PACKAGE_NAME = process.env.PLAID_ANDROID_PACKAGE_NAME || "";
 
 // We store the access_token in memory - in production, store it in a secure
 // persistent data store
-var ACCESS_TOKEN = null;
-var PUBLIC_TOKEN = null;
-var ITEM_ID = null;
+let ACCESS_TOKEN = null;
+let PUBLIC_TOKEN = null;
+let ITEM_ID = null;
 // The payment_id is only relevant for the UK Payment Initiation product.
 // We store the payment_id in memory - in production, store it in a secure
 // persistent data store
-var PAYMENT_ID = null;
+let PAYMENT_ID = null;
 
 // Initialize the Plaid client
 // Find your API keys in the Dashboard (https://dashboard.plaid.com/account/keys)
-var client = new plaid.Client({
+const client = new plaid.Client({
   clientID: PLAID_CLIENT_ID,
   secret: PLAID_SECRET,
   env: plaid.environments[PLAID_ENV],
   options: {
-    version: '2019-05-29',
+    version: "2019-05-29"
```

Comment:
should this be latest?

Reply:
O I see you're still supporting 6.

Reply:
updating to latest is a different project alltogether